polaris-react/src/components/IndexTable/IndexTable.scss

```
@@ -217,6 +217,33 @@ $loading-panel-height: 53px;
     }
   }
 
+  &.TableRow-child {
+    // stylelint-disable -- Polaris component custom properties
+    --pc-index-table-checkbox-width: var(--p-space-500);
+    --pc-index-table-checkbox-offset-left: var(--p-space-300);
+    --pc-index-table-checkbox-border-width: var(--p-border-width-0165);
+    --pc-index-table-checkbox-child-offset: calc(
+      var(--pc-index-table-checkbox-offset-left) +
+        var(--pc-index-table-checkbox-width)
+    );
+    --pc-index-table-shifted-checkbox: 32;
+
+    .TableCell-first {
+      left: var(--pc-index-table-checkbox-child-offset);
+      z-index: var(--pc-index-table-shifted-checkbox);
+      @media #{$p-breakpoints-sm-down} {
+        left: calc(var(--pc-index-table-checkbox-child-offset));
+      }
+    }
+
+    .TableCell-first + .TableCell {
+      padding-left: var(--pc-index-table-checkbox-child-offset);
+      @media #{$p-breakpoints-sm-down} {
+        padding-left: var(--pc-index-table-checkbox-child-offset);
+      }
+    }
+  }
```

It seems like `padding-left` with this value has already been applied and removing it doesn't change anything. Why is it needed twice here? Is there another way?

```suggestion
    .TableCell-first + .TableCell {
      padding-left: var(--pc-index-table-checkbox-child-offset);
    }
  }
```

https://github.com/Shopify/polaris/assets/15054990/b3391c75-4286-42ef-989d-ea3c63434464

---

Thank you for pointing on that! Seems there is no need for specifying it twice!